### PR TITLE
Update docs for SQlite

### PR DESCRIPTION
**Pull request by elegaanz**

### CODE REVIEW

File: docs/INSTALL.md [hunk 1]
```diff
@@ -79,61 +109,39 @@ export PATH="$PATH:/home/plume/.cargo/bin:/home/plume/.local/bin:/usr/local/sbin
...
+Plume needs to be compiled from source. To download the code, run:
 
 ```bash
 git clone https://github.com/Plume-org/Plume.git
 cd Plume
-
-# This may take some time as RustUp will download all
-# the required Rust components, and Cargo will download
-# and compile all dependencies.
-cargo build
```
Comment (igalic) [inline docs/INSTALL.md]: where's the build step gone to?

Comment (BaptisteGelez) [reply docs/INSTALL.md]: In `cargo install`, which installs `plume` and `plm` on the system.

File: docs/INSTALL.md [hunk 2]
```diff
@@ -79,71 +118,58 @@ export PATH="$PATH:/home/plume/.cargo/bin:/home/plume/.local/bin:/usr/local/sbin
...
-## Starting Plume
+## Running Plume
 
-First, you'll need to install Plume and the CLI tools to manage your instance.
+Then, you'll need to install Plume and the CLI tools to manage your instance.
 
 ```
-cargo install && cargo install --path plume-cli
+cargo install --no-default-features --features $FEATURES
+cargo install --no-default-features --features $FEATURES --path plume-cli
```
Comment (BaptisteGelez) [inline docs/INSTALL.md]: @igalic :arrow_up:

Comment (igalic) [reply docs/INSTALL.md]: who needs build when you can just install 😅

**Pull request merged**